polybar-scripts/system-bluetooth/system-bluetooth.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Please use `#!/bin/sh` and fix all shellcheck errors.

Reply:
It now use sh instead of bash and all shellcheck errors are fixed.